app/views/pages/feedbacks.html.erb

```diff
@@ -0,0 +1,13 @@
+<%= govuk_link_to("Home", "/") %>
+<%= govuk_label text: 'Give feedback on Request personal information', tag: 'h2', size: 'l' %>
```

Comment:
I don't think this should be a label, it's not part of the form. It should just be an H2